src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js

```diff
@@ -244,17 +249,29 @@ export default {
       if (columnName === 'LineDescription') {
         return row.lineDescription
       }
-      const currency = this.pointOfSalesCurrency.iSOCode
+      const currencyCode = this.pointOfSalesCurrency.iSOCode
       if (columnName === 'CurrentPrice') {
-        return this.formatPrice(row.priceActual, currency)
+        return this.formatPrice({
+          value: row.priceActual,
```

Comment:
Just can see change for source code with the same functionality, please let it without changes